### DISCUSSION TITLE
how to change fg of "size" and "percentage" pills?

### DISCUSSION THREAD

**Discussion by ehula** [sxyazi/yazi#1651] [1. Q&A]
It's a simple question: how do I change the fg color of the "size" and "percentage" pill areas in the status bar? It appears just to use the bg color of "mode_...".

This isn't a great solution as it severely limits the color choices for these areas while still being readable.

**Comment by sxyazi** [comment]
> It appears just to use the bg color of "mode_...".

v0.4 will separate the size and percentage colors from `mode_*`, see https://github.com/sxyazi/yazi/issues/1228 and https://github.com/sxyazi/yazi/issues/1644
